nalgebra-sparse/src/convert/serial.rs

```diff
@@ -218,7 +218,7 @@ where
 /// Converts a [`CscMatrix`] to a [`CsrMatrix`].
 pub fn convert_csc_csr<T>(csc: &CscMatrix<T>) -> CsrMatrix<T>
 where
-    T: Scalar,
+    T: Scalar + Zero,
```

Comment:
What is the reason for the additional bound?

Reply:
It was required after I introduced optional sorting in `try_from_csc_data`, now it should not be necessary.